internal/types/event.go

```diff
@@ -0,0 +1,24 @@
+package types
+
+type EventTypes string
```

Comment:
Do you think `EventType` would be better naming? i.e. singular like `EventCategory` below